go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/go-ldap/ldif
+
+go 1.14
+
+require (
+	github.com/go-asn1-ber/asn1-ber v1.4.1 // indirect
+	github.com/go-ldap/ldap v3.0.3+incompatible
+	github.com/go-ldap/ldap/v3 v3.1.7
+	gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d // indirect
```

Comment:
This line seems to be leftover from some transitive import - maybe it can be removed now?

Reply:
Go mod tidy fixed all this